main.obj : error LNK2019: 无法解析的外部符号 "bool __cdecl testing::internal::IsTrue(bool)" (?IsTrue@internal@testing@@YA_N_N@Z)，函数 "public: static void (__cdecl*__cdecl testing::internal::SuiteApiResolver<class testing::Test>::GetSetUpCaseOrSuite(char const *,int))(void)" (?GetSetUpCaseOrSuite@?$SuiteApiResolver@VTest@testing@@@internal@testing@@SAP6AXXZPBDH@Z) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: __thiscall testing::internal::GTestLog::GTestLog(enum testing::internal::GTestLogSeverity,char const *,int)" (??0GTestLog@internal@testing@@QAE@W4GTestLogSeverity@12@PBDH@Z)，函数 "public: static void (__cdecl*__cdecl testing::internal::SuiteApiResolver<class testing::Test>::GetSetUpCaseOrSuite(char const *,int))(void)" (?GetSetUpCaseOrSuite@?$SuiteApiResolver@VTest@testing@@@internal@testing@@SAP6AXXZPBDH@Z) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: __thiscall testing::internal::GTestLog::~GTestLog(void)" (??1GTestLog@internal@testing@@QAE@XZ)，函数 "public: static void (__cdecl*__cdecl testing::internal::SuiteApiResolver<class testing::Test>::GetSetUpCaseOrSuite(char const *,int))(void)" (?GetSetUpCaseOrSuite@?$SuiteApiResolver@VTest@testing@@@internal@testing@@SAP6AXXZPBDH@Z) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: __thiscall testing::Message::Message(void)" (??0Message@testing@@QAE@XZ)，函数 "private: virtual void __thiscall FooTest_HandleNoneZeroInput_Test::TestBody(void)" (?TestBody@FooTest_HandleNoneZeroInput_Test@@EAEXXZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "class testing::AssertionResult __cdecl testing::AssertionSuccess(void)" (?AssertionSuccess@testing@@YA?AVAssertionResult@1@XZ)，函数 "class testing::AssertionResult __cdecl testing::internal::CmpHelperEQ<int,int>(char const *,char const *,int const &,int const &)" (??$CmpHelperEQ@HH@internal@testing@@YA?AVAssertionResult@1@PBD0ABH1@Z) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "class testing::AssertionResult __cdecl testing::internal::EqFailure(char const *,char const *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,bool)" (?EqFailure@internal@testing@@YA?AVAssertionResult@2@PBD0ABV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@1_N@Z)，函数 "class testing::AssertionResult __cdecl testing::internal::CmpHelperEQFailure<int,int>(char const *,char const *,int const &,int const &)" (??$CmpHelperEQFailure@HH@internal@testing@@YA?AVAssertionResult@1@PBD0ABH1@Z) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "void const * __cdecl testing::internal::GetTestTypeId(void)" (?GetTestTypeId@internal@testing@@YAPBXXZ)，函数 "void __cdecl `dynamic initializer for 'private: static class testing::TestInfo * FooTest_HandleNoneZeroInput_Test::test_info_''(void)" (??__E?test_info_@FooTest_HandleNoneZeroInput_Test@@0QAVTestInfo@testing@@A@@YAXXZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "class testing::TestInfo * __cdecl testing::internal::MakeAndRegisterTestInfo(char const *,char const *,char const *,char const *,struct testing::internal::CodeLocation,void const *,void (__cdecl*)(void),void (__cdecl*)(void),class testing::internal::TestFactoryBase *)" (?MakeAndRegisterTestInfo@internal@testing@@YAPAVTestInfo@2@PBD000UCodeLocation@12@PBXP6AXXZ3PAVTestFactoryBase@12@@Z)，函数 "void __cdecl `dynamic initializer for 'private: static class testing::TestInfo * FooTest_HandleNoneZeroInput_Test::test_info_''(void)" (??__E?test_info_@FooTest_HandleNoneZeroInput_Test@@0QAVTestInfo@testing@@A@@YAXXZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: virtual __thiscall testing::Test::~Test(void)" (??1Test@testing@@UAE@XZ)，函数 "public: virtual __thiscall FooTest_HandleNoneZeroInput_Test::~FooTest_HandleNoneZeroInput_Test(void)" (??1FooTest_HandleNoneZeroInput_Test@@UAE@XZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "protected: __thiscall testing::Test::Test(void)" (??0Test@testing@@IAE@XZ)，函数 "public: __thiscall FooTest_HandleNoneZeroInput_Test::FooTest_HandleNoneZeroInput_Test(void)" (??0FooTest_HandleNoneZeroInput_Test@@QAE@XZ) 中引用了该符号
main.obj : error LNK2001: 无法解析的外部符号 "protected: virtual void __thiscall testing::Test::SetUp(void)" (?SetUp@Test@testing@@MAEXXZ)
main.obj : error LNK2001: 无法解析的外部符号 "protected: virtual void __thiscall testing::Test::TearDown(void)" (?TearDown@Test@testing@@MAEXXZ)
main.obj : error LNK2019: 无法解析的外部符号 "public: static class testing::UnitTest * __cdecl testing::UnitTest::GetInstance(void)" (?GetInstance@UnitTest@testing@@SAPAV12@XZ)，函数 "int __cdecl RUN_ALL_TESTS(void)" (?RUN_ALL_TESTS@@YAHXZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: int __thiscall testing::UnitTest::Run(void)" (?Run@UnitTest@testing@@QAEHXZ)，函数 "int __cdecl RUN_ALL_TESTS(void)" (?RUN_ALL_TESTS@@YAHXZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "void __cdecl testing::InitGoogleTest(int *,char * *)" (?InitGoogleTest@testing@@YAXPAHPAPAD@Z)，函数 _main 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: __thiscall testing::internal::AssertHelper::AssertHelper(enum testing::TestPartResult::Type,char const *,int,char const *)" (??0AssertHelper@internal@testing@@QAE@W4Type@TestPartResult@2@PBDH1@Z)，函数 "private: virtual void __thiscall FooTest_HandleNoneZeroInput_Test::TestBody(void)" (?TestBody@FooTest_HandleNoneZeroInput_Test@@EAEXXZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: __thiscall testing::internal::AssertHelper::~AssertHelper(void)" (??1AssertHelper@internal@testing@@QAE@XZ)，函数 "private: virtual void __thiscall FooTest_HandleNoneZeroInput_Test::TestBody(void)" (?TestBody@FooTest_HandleNoneZeroInput_Test@@EAEXXZ) 中引用了该符号
main.obj : error LNK2019: 无法解析的外部符号 "public: void __thiscall testing::internal::AssertHelper::operator=(class testing::Message const &)const " (??4AssertHelper@internal@testing@@QBEXABVMessage@2@@Z)，函数 "private: virtual void __thiscall FooTest_HandleNoneZeroInput_Test::TestBody(void)" (?TestBody@FooTest_HandleNoneZeroInput_Test@@EAEXXZ) 中引用了该符号
C:\Users\wisechip\Desktop\workspace\src\train\third_party\gtest\debug\lib\gtest.lib : warning LNK4272: 库计算机类型“x64”与目标计算机类型“x86”冲突
C:\Users\wisechip\Desktop\workspace\src\train\Debug\stardard_library.exe : fatal error LNK1120: 18 个无法解析的外部命令
